VultisigApp/VultisigApp/Views/New Wallet/CreateVaultView.swift

```diff
@@ -73,3 +64,22 @@ struct CreateVaultView: View {
 #Preview {
     CreateVaultView()
 }
+
+#if os(macOS)
+extension CreateVaultView {
+    var main: some View {
+        VStack {
+            headerMac
```

Comment:
I would suggest file structure like: 
CreateVaultView.swift
CreateVaultView+iOS.swift
CreateVaultView+MacOS.swift

For ex CreateVaultView+MacOS.swift can looks like: 

```swift
extension CreateVaultView {
    var main: some View {
        VStack {
            headerMac
            ...

 /// All MacOS specific views also here
 var headerMac: some View {
       ...
 }
```